main/ui/ui.go

```diff
@@ -84,6 +84,16 @@ func main() {
 		TimeToLive:   time.Minute * 5, // Cache items invalidated after 5 minutes.
 		RequestLimit: 500,
 	})
+	for i := 0; i < 10; i++ {
```

Comment:
In addition, any client using MQE as a library (which is probably a lot more likely, in order to get it to integrate with other systems better) wouldn't be using this either.